lib/packages/core/src/api/factories/message/sendTextMessage.ts

```diff
@@ -29,7 +30,7 @@ export const sendTextMessage = (service: BurstService):
         recipientId: string,
         senderPublicKey: string,
         senderPrivateKey: string,
-        deadline: number = 1440,
+        deadline: number = DefaultDeadline,
         fee: number = 0.1,
```

Comment:
could be defaulted also, or turned into a required parameter (no default argument)